worker/raft/worker.go

```diff
@@ -178,13 +197,17 @@ func newWorker(config Config) (*Worker, error) {
 	if err := config.Validate(); err != nil {
 		return nil, errors.Trace(err)
 	}
+	if config.NoLeaderTimeout == time.Duration(0) {
```

Comment:
Don't need the cast

Reply:
Oh, nice! I'm not really clear on when they're needed and when they're not. I'll remove.